byoqm/source_coordinator/source_coordinator.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, src_root: Path, langauge: str):
 
         self.language = Language(_TREESITTER_BUILD.__str__(), langauge)
         self.asts = {}
-
+        self.prog_lang = langauge
```

Comment:
To me I'd expect `language` to already be a programming language, so maybe we should promote `prog_lang` to the name `language` and demote the current `language` to the name `tree_sitter_language`?

Reply:
I would maybe argue that both should be more specifically defined?

Reply:
I agree that generally speaking as programmers, when we talk about languages, we are addressing which programming language is in use.